asnake/utils/__init__.py

```diff
@@ -56,8 +56,7 @@ def parse_subnote(subnote):
             data.append(item["value"])
         content = data
     else:
-        subnote_content_list = list(parse_subnote(sn)
-                                    for sn in note["subnotes"])
+        subnote_content_list = list(parse_subnote(sn) for sn in note["subnotes"])
```

Comment:
I'm sorry that I didn't think of this before, but it occurs to me: why isn't this a list comprehension rather than a generator comprehension passed into the list constructor?

```python
subnote_content_list = [parse_subnote(sn) for sn in note["subnotes"]]
```

Should be equivalent, right?

Reply:
Yep, you're right. Guess we just missed this.